cpu/esp32/periph/rtt_hw_rtc.c

```diff
@@ -74,18 +74,18 @@ static void _rtc_init(void)
 static void _rtc_poweron(void)
 {
     /* route all interrupt sources to the same RTT level type interrupt */
-    intr_matrix_set(PRO_CPU_NUM, ETS_RTC_CORE_INTR_SOURCE, CPU_INUM_RTC);
+    intr_matrix_set(PRO_CPU_NUM, ETS_RTC_CORE_INTR_SOURCE, CPU_INUM_RTT);
 
     /* set interrupt handler and enable the CPU interrupt */
-    xt_set_interrupt_handler(CPU_INUM_RTC, _rtc_isr, NULL);
-    xt_ints_on(BIT(CPU_INUM_RTC));
+    xt_set_interrupt_handler(CPU_INUM_RTT, _rtc_isr, NULL);
```

Comment:
For this file only the define `CPU_INUM_RTC` is renamed to keep it compilable. The changes for ESP-IDF for interrupt handling for this file are in commit 6cca58a6d1f15b589312c80e8a47c85bb59546c5.